### PR TITLE
fix math comments

### PR DESCRIPTION
**Pull request by stanleyljl**
Re: #

Changelog:
 * fix math comments

### CODE REVIEW

File: cocos/core/math/utils.ts [hunk 1]
```diff
@@ -245,8 +245,8 @@ export function inverseLerp (from: number, to: number, value: number) {
 /**
  * @en Compare the absolute values of all components and the component with the largest absolute value will be returned.
  * @zh 对所有分量的绝对值进行比较大小，返回绝对值最大的分量。
- * @param v 类 Vec3 结构
- * @returns 绝对值最大的分量
+ * @param v vec3 like value
+ * @returns max absolute component
```
Comment (minggo) [inline cocos/core/math/utils.ts:247]: It is better to keep both English and Chinese.

File: cocos/core/value-types/value-type.ts [hunk 2]
```diff
@@ -59,8 +59,8 @@ export class ValueType {
      * @zh
      * 判断当前值是否与指定值相等。此判断应当具有交换性，即满足 `this.equals(other) === other.equals(this)`。
      * 本方法的基类版本简单地返回 `false`。
-     * @param other 相比较的值。
-     * @returns 相等则返回 `true`，否则返回 `false`。
+     * @param other The other object
+     * @returns `true` if equal, otherwise returns `false`
```
Comment (minggo) [inline cocos/core/value-types/value-type.ts:62]: The same as above.

**Comment by stanleyljl** [comment]
skip ci

File: cocos/core/math/utils.ts (hunk 1 above)
Comment (stanleyljl) [reply cocos/core/math/utils.ts:247]: A large number of parameters are lack of Chinese notes, so it is suggested to change them uniformly later.

**Review by minggo — APPROVED** [review]

**Comment by pandamicro** [comment]
https://github.com/cocos/cocos-engine/issues/10840

**Pull request merged**